lib/CMakeLists.txt

```diff
@@ -98,7 +105,7 @@ else()
 endif()
 
 if (NOT DEFINED LLVM_TARGETS_TO_BUILD)
-  set(LLVM_TARGETS_TO_BUILD X86)
+  set(LLVM_TARGETS_TO_BUILD "X86;ARM")
```

Comment:
What sets this?

Reply:
This comes from `llvm_archs` in the Makefile.

Reply:
Gotcha